src/components/mypage/ReviewModal/index.tsx

```diff
@@ -0,0 +1,135 @@
+import type { ChangeEventHandler, ReactElement } from 'react'
+import { useState } from 'react'
+import { Styled } from './styled'
+import type {
+  ReviewModalProps,
+  EvaluateState,
+  EvaluateData,
+  ReviewState
+} from './types'
+
+const evaluateData: EvaluateData = [
+  {
+    state: 'smile',
+    text: '좋아요'
+  },
+  {
+    state: 'meh',
+    text: '보통이에요'
+  },
+  {
+    state: 'sad',
+    text: '별로에요'
+  }
+]
+
+const ReviewModal = ({
+  isOpen = true,
+  onClose,
+  onClick,
+  nickName = '닉네임',
+  productName = '상품이름',
+  isReadMode = false,
+  readModeEvaluate = 'smile',
+  readModeReviewContent = '리뷰'
```

Comment:
```suggestion
  score = 'smile',
  contents = '리뷰'
```
이미 컨텍스트가 포함되어 있어서 props명에서 readMode를 생략하고 타입으로 isReadMode가 true일 때만 해당 props를 받을 수 있는 형태가 좋을 것 같습니다!

Reply:
완료!
[링크](https://github.com/price-offer/offer-fe/pull/108/commits/a3f77e04b05fcd12a81119f976d0dc278ed42b34#diff-4b8ae2495983bb2ac6378048f00a9113a977027de98963d259a27f5437cb9424)

Reply:
@chojooyoung 넴 요기 `타입으로 isReadMode가 true일 때만 해당 props를 받을 수 있는 형태`가 반영이 안되어있는 것 같아요!